parsec/CMakeLists.txt

```diff
@@ -234,11 +242,118 @@ if( BUILD_PARSEC )
   endif(CMAKE_Fortran_COMPILER_SUPPORTS_F90)
 
   install(TARGETS parsec
-    EXPORT parsec-targets
-    ARCHIVE DESTINATION ${PARSEC_INSTALL_LIBDIR}
-    LIBRARY DESTINATION ${PARSEC_INSTALL_LIBDIR}
-    PUBLIC_HEADER DESTINATION ${PARSEC_INSTALL_INCLUDEDIR})
+          EXPORT parsec-targets
+          ARCHIVE DESTINATION ${PARSEC_INSTALL_LIBDIR}
+          LIBRARY DESTINATION ${PARSEC_INSTALL_LIBDIR})
+  # Cmake < 3.20 does not support hierarchical PUBLIC and PRIVATE _HEADER,
```

Comment:
Maybe it could make sense to require cmake >= 3.20, we are already at 3.18 so its a small step.